packages/terra-form-checkbox/src/CheckboxField.jsx

```diff
@@ -113,7 +113,15 @@ const CheckboxField = (props, { intl }) => {
     <legend id={legendAriaDescriptionId} className={cx(['legend-group', { 'legend-group-hidden': isLegendHidden }])}>
       <div {...legendAttrs} className={legendClassNames}>
         {isInvalid && <span className={cx('error-icon')} />}
-        {required && (isInvalid || !hideRequired) && <span className={cx('required')}>*</span>}
+        {required && (isInvalid || !hideRequired) && (
+          <React.Fragment>
+            <div aria-hidden="true" className={cx('required')}>*</div>
+            <span className={cx('visually-hidden-text')}>
+              {intl.formatMessage({ id: 'Terra.form.field.required' })}
+              {' '}
+            </span>
```

Comment:
want to use terra-visually-hidden-text instead?

Reply:
Good call, updated in this commit: https://github.com/cerner/terra-core/pull/2398/commits/327cf153bdc80315fc4eb6126429b7be303b9b6b